### PR TITLE
fix: loading fix for adv search and some other minor fixes

### PR DESCRIPTION
**Pull request by tomaszandel**
- loading fixed 
- providers default fixed
- paging fixed

### CODE REVIEW

**Comment by Marcelinna** [comment]
@tomaszandel 
Providers
image

**Comment by Marcelinna** [comment]
1. Go to 6 page in Datasource collection
2. Type "datasource" in main serach
image
3. Click enter
4. See problem
image

**Comment by Marcelinna** [comment]
@tomaszandel Please check if Publisher for Other is correct?
image

File: ui/apps/ui/src/app/components/search-input/search-input.component.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class SearchInputComponent implements OnInit {
   );
 
   collectionFcAdvForm = new FormControl<ICollectionTagsConfig>(
-    this.collectionFcAdv[0],
+    this.collectionFcAdv[2],
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/components/search-input/search-input.component.ts:92]: What exactly does it fix? 
init value from  `{ name: 'Author' },`to  `{ name: 'In title' }`
Later on I see: `this.collectionFcAdvForm.reset(this.collectionFcAdv[2]);` 
The init value was the issue why searching on page 6 wasn't working?
Maybe place some comment about it in the code. Its kinda a magical number

File: ui/apps/ui/src/app/collections/data/other-resources-products/filters.data.ts [hunk 2]
```diff
@@ -119,7 +119,7 @@ export const otherResourcesProductsFilters: IFiltersConfig = {
       id: 'publisher',
       filter: 'publisher',
       label: 'Publisher',
-      type: 'tag',
+      type: 'multiselect',
       defaultCollapsed: false,
```
Comment (Michal-Kolomanski) [inline ui/apps/ui/src/app/collections/data/other-resources-products/filters.data.ts:123]: rather true?

**Comment by Marcelinna** [comment]
@tomaszandel Please make filter Publisher collapsed

**Review by Michal-Kolomanski — APPROVED** [review]
LGTM, only minors.

Please squash commits.
Just a friendly reminder for future reference: It would be great if you could separate each fix into a specific commit with an appropriate commit message :D

File: ui/apps/ui/src/app/components/search-input/search-input.component.ts (hunk 1 above)
Comment (tomaszandel) [reply ui/apps/ui/src/app/components/search-input/search-input.component.ts:92]: itsa. long story, but its not related to search, its related to non existing provider search forms, i choosen the one that exist on both sides

File: ui/apps/ui/src/app/components/filters/utils.ts [hunk 3]
```diff
@@ -36,45 +36,52 @@ export function* toAllLevels(value: string) {
...
-    }))
-    .reduce((acc, node) => ({ ...acc, ...node }), {});
-  const idMap = nodes
-    .map((node) => ({
-      [node.id]: { ...node, children: [] } as IUIFilterTreeNode,
-    }))
-    .reduce((acc, node) => ({ ...acc, ...node }), {});
-  const fullMap = { ...allLvlsPermIdMap, ...idMap };
-  const allNodes = Object.values(fullMap);
+  if (nodes[0].filter !== 'publisher') {
```
[the remark this answers is not in the document]
Comment (tomaszandel) [reply ui/apps/ui/src/app/components/filters/utils.ts:39]: its only for publisher

File: ui/apps/ui/src/app/collections/services/fetch-data.service.ts [hunk 4]
```diff
@@ -59,7 +59,9 @@ export class FetchDataService {
       )
       .pipe(
         catchError(() => of(_EMPTY_RESPONSE)),
-        tap(() => this._paginationRepository.setLoading(false)),
+        tap(() => {
+          // this._paginationRepository.setLoading(false);
```
[the remark this answers is not in the document]
Comment (tomaszandel) [reply ui/apps/ui/src/app/collections/services/fetch-data.service.ts:63]: still usable tap, as we plan to refactor i would rather keep it for now

File: ui/apps/ui/src/app/collections/data/bundles/adapter.data.ts [hunk 5]
```diff
@@ -12,9 +12,7 @@ import {
 
 function getBundleUrl(bundle: Partial<IBundle> & { id: string }): string {
   if (bundle.iid != null && bundle.iid.length === 1) {
-    return `${ConfigService.config?.marketplace_url}/services/${
-      bundle.service_id
-    }/bundles/${bundle.iid![0]}`;
+    return `${ConfigService.config?.marketplace_url}/services/${bundle.service_id}/bundles/${bundle.iid?.[0]}`;
```
[the remark this answers is not in the document]
Comment (tomaszandel) [reply ui/apps/ui/src/app/collections/data/bundles/adapter.data.ts:15]: its a linter error fix ;) based on other declarations

**Review by Marcelinna — APPROVED** [review]

File: docker-compose.yml [hunk 6]
```diff
@@ -10,7 +10,7 @@ services:
       - ess-local
     environment:
       DATABASE_URI: postgresql+psycopg2://${DB_POSTGRES_USER:-ess}:${DB_POSTGRES_PASSWORD:-ess}@db:5432/${DB_POSTGRES_DB:-ess}
-      SOLR_URL: http://149.156.182.2:8983/solr/
+      SOLR_URL: http://149.156.182.69:8983/solr/
```
Comment (Michal-Kolomanski) [inline docker-compose.yml:13]: I am not convinced that we should connect to the production solr instance by default. Its risky. I would prefer to keep beta instance as it was

**Pull request merged**